### PR TITLE
AL-2211: Feat desk booking + Vercel deployment 

### PR DESCRIPTION
**Pull request by bnjm**
- adds support for booking desks as well as rooms (509d0fefea89c7887bc37a2070863bb1369054f4)
- fixes a bug in the existing app when highlighted state gets reset (daa90ab9f588c5dd0a37d355ef1875be90b7f691)
- replaces the server with a serverless endpoint (d2f8d397d08c5907f37a7c0c3aaf98d3823ee791 + readme changes: c1aa4d3321c15fe7cd20d56f435c67adb8f410e9)
- uses [Vercel](https://vercel.com/) for deployment + local dev

### CODE REVIEW

File: src/reducers/bookings.ts [hunk 1]
```diff
@@ -208,12 +227,12 @@ const aggregateUsedSlots = (
   })
 }
 
-// calculate used spaces at a especific moment
-const aggregateUsedSpaces = (
+// calculate used items at a especific moment
```
Comment (manuatarchilogic) [inline src/reducers/bookings.ts:230]:
```suggestion
// calculate used items at a specific moment
```

that was probably written by a Spanish speaking person :) (LagarSoft is Argentinia based?)

File: src/App.tsx [hunk 2]
```diff
@@ -61,28 +63,32 @@ const App = (props: PropsFromRedux) => {
...
+    if (isBookRoomModalVisible === false) props.unSelectItem()
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isBookRoomModalVisible])
 
   return (
     <div className="app">
-      <div className="header">Room Booking - {props.floorName}</div>
+      <div className="header">
+        Desk & Room Booking
+        <div className="subheader">{props.floorName}&nbsp;</div>
```
Comment (manuatarchilogic) [inline src/App.tsx:79]: what's up with that non-breaking space?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
 
-## Setup local server
 
-Navigate to the server folder and run:
+## Set Environment Variables
 
-    yarn install
+To run the app we'll need to set some environment variables first.  
+
+- The backend needs a 'secret token' to negotiate a temporary token: secret token: [secret token documentation](https://developers.archilogic.com/space-api/v2/introduction.html#secret-access-token) documentation
```
Comment (manuatarchilogic) [inline README.md:12]:
```suggestion
- The backend needs a 'secret token' to negotiate a temporary token: [secret token documentation](https://developers.archilogic.com/space-api/v2/introduction.html#secret-access-token)
```

**Review by manuatarchilogic — APPROVED** [review]
nice! thanks for putting in the work

it was probably already broken, but the the form fields in the booking dialog (first/last name, email) are not correctly set and reset when you book or view a booking

File: README.md (hunk 3 above)
Comment (bnjm) [reply README.md]: [42e57e9](https://github.com/archilogic-com/room-booking/pull/26/commits/42e57e9f9c023cff8dc4b4cfacf59582612cace0)

File: src/reducers/bookings.ts (hunk 1 above)
Comment (bnjm) [reply src/reducers/bookings.ts]: [42e57e9](https://github.com/archilogic-com/room-booking/pull/26/commits/42e57e9f9c023cff8dc4b4cfacf59582612cace0)

**Comment by manuatarchilogic** [comment]
how do we access to Vercel?

File: src/App.tsx (hunk 2 above)
Comment (bnjm) [reply src/App.tsx]: It was just a lazy way of preventing content-shift when `floorName` loads, now *slightly* less lazy: [dc81306](https://github.com/archilogic-com/room-booking/pull/26/commits/dc81306b28cc9bd1a2ac6e5b20e2c7d009ca3145)

**Pull request merged**